src/network/network_client.ml

```diff
@@ -51,4 +51,8 @@ let broadcast_to_list (type req res)
   uris
   (* TODO: limit concurrency here *)
   |> Lwt_list.iter_p (fun uri ->
-         Lwt.catch (fun () -> raw_post E.path data uri) (fun _exn -> await ()))
+         Lwt.catch
+           (fun () -> raw_post E.path data uri)
+           (fun exn ->
+             Log.error "Broadcast exception: %a\n%!" Fmt.exn exn;
```

Comment:
We may want to add more information here? e.g., but not necessarily as such:
```ocaml
let endpoint = E.path in
let uri_s = Uri.to_string uri in
Log.error "Broadcast exception: %a when sending %s to %s\n%!" Fmt.exn exn endpoint uri_s;
```

Reply:
Done.